### PR TITLE
Upgrade Vello and ditch SceneBuilder/SceneFragment.

### PR DESCRIPTION
**Pull request by xorgy**

### CODE REVIEW

**Comment by xorgy** [comment]
This is actually busted, I'll try to fix it.

**Comment by dfrg** [comment]
oops, I did the same PR :)

File: src/app_main.rs [hunk 1]
```diff
@@ -218,8 +217,7 @@ where
             } else {
                 None
             };
-            let mut builder = SceneBuilder::for_scene(&mut self.scene);
-            builder.append(fragment, transform);
+            self.scene.append(fragment, transform);
```
Comment (dfrg) [inline src/app_main.rs:220]: `SceneBuilder::for_scene` implicity reset the scene. We'll have to do this manually now: `self.scene.reset()`

File: src/widget/core.rs [hunk 2]
```diff
@@ -446,28 +448,26 @@ impl Pod {
         };
 
         if needs_paint {
-            let mut builder = SceneBuilder::for_fragment(&mut self.fragment);
-            self.widget.paint(&mut inner_cx, &mut builder);
+            self.widget.paint(&mut inner_cx, &mut self.fragment);
```
Comment (dfrg) [inline src/widget/core.rs:451]: ditto, `self.fragment.reset()`

Comment (xorgy) [reply src/widget/core.rs]: Yeah I noticed quickly. Even after closing the examples my GPU driver crashed my system about 15 seconds later... I missed it in core and made the same mistake. :grimacing: 
Anyway, resolved.

**Pull request merged**